StPicoEvent/StPicoBTofPidTraits.cxx

```diff
@@ -1,3 +1,13 @@
+/***************************************************************************
+ *
+ * $Id$
+ *
+ * Author: Xin Dong (xdong@lbl.gov)
```

Comment:
I will take your silence on this issue as the go ahead to ignore the guidelines ;-)
Since both, maintainers and reviewer(s), agree that it is unnecessary, maybe we can make a case.

I think Jerome, would ask to add the GetCVS line which will force us to have CVS $log and $id. I think this whole thing is needed so we can have the version id of the code in the log files.